configuration/debian/tedge_agent/postrm

```diff
@@ -26,14 +9,7 @@ purge_agent_directory() {
 
 case "$1" in
     purge)
-        remove_tedge_agent_user
-        remove_sudoers_file
-        purge_agent_directory
-    ;;
-
-    remove)
```

Comment:
You need to keep this line
```
remove)
;;
```

Otherwise, it will drop to the case `*)`, but `remove` shouldn't be an unknown argument.